Formula/bareos-fd.rb

```diff
@@ -0,0 +1,65 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/16.2.7.tar.gz"
+  sha256 "0b8dbe2e17b3eda470c30d4fe6af92e6f7b668fd1cfc1b045becdc8c8b6a767f"
+
+  depends_on "readline"
+  depends_on "openssl"
+  depends_on "python"
```

Comment:
`depends_on :python if MacOS.version <= :snow_leopard` usually rather than `"python"`, and even then, only if it absolutely needs Python 2.7.